client-keycloak/src/main/java/org/apache/atlas/keycloak/client/service/AtlasKeycloakAuthService.java

```diff
@@ -89,14 +88,12 @@ public String getAuthToken() {
     }
 
     public boolean isTokenExpired() {
-        return (currentTime() + DEFAULT_MIN_VALIDITY_MINS) >= expirationTime;
```

Comment:
why DEFAULT_MIN_VALIDITY_MINS is removed?

Reply:
its not required, we are taking care in  during `expirationTime`  itself. We are doing -10mins and setting the expiry time.